src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useContext, FunctionComponent } from 'react';
+import { Omit } from 'utility-types';
+
+import PaymentContext from '../PaymentContext';
+
+import HostedWidgetPaymentMethod, { HostedWidgetPaymentMethodProps } from './HostedWidgetPaymentMethod';
+// import {WithPaymentProps} from "../withPayment";
+
+export type PaypalCommercePaymentMethod = Omit<HostedWidgetPaymentMethodProps, 'containerId'>;
+
+const PaypalCommercePaymentMethod: FunctionComponent<PaypalCommercePaymentMethod> = ({
+      initializePayment,
+      onUnhandledError,
+      hidePaymentButton,
+      submitForm,
```

Comment:
You can do `connectFormik(PaypalCommercePaymentMethod)`, which injects `formik.submitForm` into the component. And you can use it directly without having to pass `submitForm` all the way down from `PaymentForm`.

Reply:
You are right! Thank you!